An error has occurred while creating the directory defined by projectBuildDir: path

OS: Windows Home 10
IDE: Intellij IDEA COMMUNITY 2019.2
MAVEN: 3.6.2
JDK: 1.8

Vertx: 3.8.2

maven plugin:
``` xml

<build>
        <plugins>
            <plugin>
                <groupId>io.reactiverse</groupId>
                <artifactId>vertx-maven-plugin</artifactId>
                <version>1.0.21</version>
                <executions>
                    <execution>
                        <id>vmp</id>
                        <goals>
                            <goal>initialize</goal>
                            <goal>package</goal>
                        </goals>
                    </execution>
                </executions>
                <configuration>
                    <redeploy>true</redeploy>
                    <verticle>com.xxx.sequence.MainVerticle</verticle>
                    <launcher>com.xxx.sequence.SequenceLauncher</launcher>
                </configuration>
            </plugin>
        </plugins>
    </build>

```

terminal output:

``` 

[INFO] --- maven-compiler-plugin:3.1:testCompile (default-testCompile) @ sequence ---
[INFO] Not compiling test sources
[INFO]
[INFO] --- maven-surefire-plugin:2.12.4:test (default-test) @ sequence ---
[INFO] Tests are skipped.
[INFO]
[INFO] --- maven-jar-plugin:2.4:jar (default-jar) @ sequence ---
[INFO] Building jar: D:\Workspace\xxx\sequence\target\sequence-1.0-SNAPSHOT.jar
[INFO]
[INFO] --- vertx-maven-plugin:1.0.21:package (vmp) @ sequence ---
[ERROR] An error has occurred while creating the directory defined by projectBuildDir: D:\Workspace\xxx\sequence\target
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  3.488 s
[INFO] Finished at: 2019-10-25T09:32:00+08:00
[INFO] ------------------------------------------------------------------------

D:\Workspace\xxx\sequence>


```

--- a/src/main/java/io/reactiverse/vertx/maven/plugin/mojos/PackageMojo.java
+++ b/src/main/java/io/reactiverse/vertx/maven/plugin/mojos/PackageMojo.java
@@ -132,9 +132,9 @@ public class PackageMojo extends AbstractVertxMojo {
 
         File jar;
         try {
-            
+
             File buildDir = new File(projectBuildDir);
-            if (!buildDir.mkdirs()) {
+            if (!buildDir.isDirectory() && !buildDir.mkdirs()) {
                 getLog().error("An error has occurred while "
                         + "creating the directory defined by projectBuildDir: " + projectBuildDir);
             }

